.ci/skipped_notebooks.yml

```diff
@@ -531,4 +531,10 @@
         - ubuntu-20.04
         - ubuntu-22.04
         - windows-2019
-
+- notebook: notebooks/llm-agent-react/llm-agent-rag-llamaindex.ipynb
```

Comment:
@OpenVINO-dev-contest  is it possible to setup replacement with some smaller model for tests only? e.g. tiny-llama or qwen2 (we do not need to check generation quality, more about checking that notebook is functional